bin/configure-scripts/deploy_kamal_instructions.rb

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
+
+require "#{__dir__}/utils"
+
+announce_section "Add 'Deploy to Kamal' instructions?"
+
+instructions_file = "#{__dir__}/deploy-instructions/kamal.md"
+
+add_instructions = ask_boolean "Would you like to add Kamal deploy instructions to your project.", "y"
+if add_instructions
+  File.open("README.md", "a") do |readme|
+    instruction = File.read(instructions_file)
+    instruction.each_line do |line|
+      readme << line
+    end
+  end
+else
+  puts "Not adding 'Deploy to Kamal' instructions.".yellow
+end
```

Comment:
We could probably add many more questions here, like "do you want to set up Kamal now?" and then prompt for the IP, etc. and auto-populate many more things.

Reply:
Yeah, that would be the ideal experience. Though, this feels like quite some extra work and we don't know whether Kamal will be the go-to option for the time to come. But yeah, the instructions suck. 😅